docs/overview_rs.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+---
+
+# Overview_RS
+
+The To-Do service is a cloud-based platform that offers subscribers a task list 
+where they can post tasks and receive timely reminders. Our API service emulates the REST 
```

Comment:
```suggestion
where they can post tasks and receive timely reminders. Our service provides the REST 
```

The Service provides the API for other applications and services to use to access the features of the service.